cmd/adapter/main.go

```diff
@@ -200,14 +200,7 @@ func runScaledObjectController(ctx context.Context, mgr manager.Manager, scaleHa
 // generateDefaultMetricsServiceAddr generates default Metrics Service gRPC Server address based on the current Namespace.
 // By default the Metrics Service gRPC Server runs in the same namespace on the keda-operator pod.
 func generateDefaultMetricsServiceAddr() string {
-	const defaultNamespace = "keda"
```

Comment:
Why are we removing this?

Reply:
Ok, I did not see the inlined method call in sprintf which is in pkg/util/env_resolver.go